### DISCUSSION TITLE
Attribute Update after creation

### DISCUSSION THREAD

**Discussion by iamdineshkumar** [appwrite/appwrite#2910] [Q&A]
In recent version, After creating attribute its not able to update. Only delete option is provided. This actually have an impact if need to change certain things for the attribute for an example make it as required or in case of enum include more element as options. But its not able to achive right now since its not editable.

**Comment by stnguyen90** [comment]
It might be good to close this and add your feedback to https://github.com/appwrite/appwrite/discussions/2720#discussioncomment-2089980

**Reply by Gizmo1337** [reply]
Had this happened to me, I had to connect to DB and find the attributes table and modify that. Imagine having a prod app and having 100k documents and needing to change an attribute size.

**Comment by joeyouss** [comment]
Hi! Since a similar comment exists, I am going to close this as mentioned by Steven. Thank you!
